src/lib.rs

```diff
@@ -570,8 +569,8 @@ impl Inner {
     }
 }
 
-unsafe fn cancel(handle: &AsRawHandle,
-                 overlapped: &windows::Overlapped) -> io::Result<()> {
+unsafe fn cancel<T: AsRawHandle>(handle: &T,
```

Comment:
This would be a change in API. Would it be possible to change the type instead to `&dyn T` to match the previous behaviour?

Reply:
`cancel` doesn't seem to be part of the public API, so I'm not sure I follow.

Reply:
Hum, so it is. Late night code reviews, huh? :smile: